cluster/ci/config/prow/config.yaml

```diff
@@ -802,3 +802,95 @@ periodics:
     - name: token
       secret:
         secretName: openshift-bot-token
+- name: periodic-issue-close
+  interval: 3h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: docker-registry.default.svc:5000/ci/commenter:latest
+      args:
+      - |-
+        --query=org:openshift
```

Comment:
This means issue closing will work across openshift, not just origin. Any objections?

Reply:
We'll hear about them soon enough :)